webapp/src/js/components/Map/Widget.js

```diff
@@ -23,41 +23,48 @@ import _max from 'lodash/max';
 import _min from 'lodash/min';
 
 // CSS
-import 'leaflet/dist/leaflet.css';
+// FIXME: leaflet/dist/leaflet.css
+/*
+ERROR in ./~/css-loader!./~/postcss-loader!./~/leaflet/dist/leaflet.css
```

Comment:
This will be fixed in the next release of leaflet: https://github.com/Leaflet/Leaflet/issues/4849